test/functional/helpers.lua

```diff
@@ -109,6 +109,10 @@ function module.request(method, ...)
   return rv
 end
 
+function module.request_lua(method, ...)
+  return module.exec_lua(string.format([[return vim.api.%s(...)]], method), ...)
```

Comment:
instead of using stringformatting we can use `module.exec_lua([[return vim.api[...](select(2, ...))]], method, ...)`